addons/website_slides/models/slide_channel_invite.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+import re
+
+from email.utils import formataddr
+
+from odoo import api, fields, models, tools, _
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+emails_split = re.compile(r"[;,\n\r]+")
+
+
+class SlideChannelInvite(models.TransientModel):
+    _name = 'slide.channel.invite'
+    _description = 'Channel Invitation Wizard'
+
+    @api.model
+    def _get_default_from(self):
+        if self.env.user.email:
+            return formataddr((self.env.user.name, self.env.user.email))
+        raise UserError(_("Unable to post message, please configure the sender's email address."))
+
+    @api.model
+    def _get_default_author(self):
+        return self.env.user.partner_id
+
+    # composer content
+    subject = fields.Char('Subject')
+    body = fields.Html('Contents', default='', sanitize_style=True)
+    attachment_ids = fields.Many2many(
+        'ir.attachment', 'slide_channel_mail_compose_message_ir_attachments_rel', 'wizard_id', 'attachment_id',
```

Comment:
Small note: I think we can either rely on orm standard table name and not give it, either give something shorter ^^

Reply:
Done. (also done for partner_ids)